doc/man7/property.pod

```diff
@@ -87,7 +120,31 @@ In order to permit a more concise expression of boolean properties, there
 is one short cut: a property name alone (e.g. "default") is
 exactly equivalent to "default=yes" in both definitions and queries.
 
-=head1 Syntax
+=head2 Global and Local
+
+Two levels of property query are supported.
+A context based property query that applies to all fetch operations and a local
+property query.
+Where both the context and local queries include a clause with the same name,
+the local clause is used and the context one ignored.
+For example, a context property query of "fips=yes" and a local property query
+of "fips=no" would result in algorithms that have the "fips" property set t
+"no".
+
+=head2 Override
+
+It is possible for a local property query to override a clause in the context
+property query by preceeding the property name with a '-'.
```

Comment:
I would not call this an override, it's more like erasing/removing the context clause.

Also, using the `-` prefix does not seem very intuitive to me. The `-fips` clause could easily be misinterpreted as "disable fips" instead of "I don't care about fips". How about adding a new special keyword instead, something like `fips=any`, `fips=dontcare`  or `fips=undef`?

Reply:
I've changed it to _remove_.

I'm less sure about the fips=xxx suggestion.  There is another prefix operator but no predefined values that do special things.  Removing one prefix operator and putting a special value in its place seems to complicated things.